app/models/approval_request.rb

```diff
@@ -4,4 +4,11 @@ class ApprovalRequest < ApplicationRecord
   enum :state => [:undecided, :approved, :denied]
 
   belongs_to :order_item
+
+  scope :by_owner, lambda {
+    select("approval_requests.*, order_items.order_id, orders.owner")
```

Comment:
Do we actually need to return the `order_id` and the `owner` as part of the request?  In this case  I believe we would just need the ActiveRecord::Relation of all `approval_requests` for a specific `owner`, but not return the owner as that can be inferred internally

Reply:
you're right we don't need those extra columns - I'll strip out the `select` part of the scope.